.pre-commit-config.yaml

```
@@ -1,27 +1,30 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 22.6.0
     hooks:
       - id: black
         language_version: python3.8
 
   - repo: https://github.com/PyCQA/isort
-    rev: 5.7.0
+    rev: 5.10.1
     hooks:
       - id: isort
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: "v0.800"
+    rev: v0.961
     hooks:
       - id: mypy
+        additional_dependencies:
+          - types-requests
+          - types-toml
```

If #43 is merged, the this should change to tomli.

```suggestion
          - tomli
```